### DISCUSSION TITLE
cannot change "tsample" as it reverts back to 600

### DISCUSSION THREAD

**Discussion by turbolooser** [revk/xESP32-Faikout#540] [General]
I have 3 Faikin running and it seems that nor via mqtt or directly on the webui I can change the "tsample" back to its default which I think was 900. It always reverts back to 600 in my case on all 3 faikins.

Any possible reason or debugging that I could run?

**Comment by revk** [comment]
I have never seen that before!
Works for me

<img width="322" alt="Screenshot 2024-10-28 at 08 46 35" src="https://github.com/user-attachments/assets/1471c0d3-07f3-4e7d-8e8a-bb2dcb68ae03">


Have you tried just changing it, e.g.

```
setting/Yourfaikin/tsample 900
```

or

```
setting/Yourfaikin {"tsample":900}
```

Does it report back any sort of error via MQTT?

**Reply by turbolooser** [reply]
I do see the same message above (change tsample) but after reboot and checking on the web interface its still the old value of "600". I also do see currently outside temp missing in mqtt in HA for all 3 Faikins ... so something is going bad here.
I have even powered the outside unit off to check for the tsample issue but now the outside temps are gone :-(

**Reply by turbolooser** [reply]
no error on the mqtt message publish.
I do see the webui as your screenshot above...then the faikin reboots and once back it still has 600 as value.
Re-running the command will say the same messages (UI) and then reboot again etc....

**Reply by revk** [reply]
OK no clue at all then, only thing I can suggest is a factory reset.

It is in the manual, and is something like `command/Yourfaikin/factory [MAC]Faikin`

"This does a factory reset of all settings, the argument has to be a string of the MAC address and the app name, e.g. 112233445566GPS"

**Reply by turbolooser** [reply]
restarting HA seem to have helped to get the outside temp reporting back to normal on mqtt.
Unfortunately the tsample issue still remains. I can run the mqtt command with 600 correctly and without a reboot (as its value is unchanged) so the command should be fine.
But when i set the a value to anything else than 600 after the reboot its still 600

**Reply by revk** [reply]
Yeh try a factory reset of the Faikin

**Reply by turbolooser** [reply]
so like this?

command/faikin-wohnzimmer/factory 308398D3D814faikin-wohnzimmer

(not sure what you mean by GPS in your reset value. Is it the Faikin name or which app name are you referring to?)

**Reply by turbolooser** [reply]
ah got it... in my case the app name is: Faikin
command/faikin-wohnzimmer/factory 308398D3D814Faikin

**Reply by revk** [reply]
Yeh

**Comment by turbolooser** [comment]
I guess I have found it.... aftzer the factory reset and without the mqtt host config in place I was able to change the tsample value correctly. When enabling the mqtt host it was running sideways again.
After deleting all kind of topics for the Faikins on the mqtt host it seems that I am now fine again 👍 

So it seems there was a weird topic/value that was "preventing" the new value...makes sense?

**Comment by revk** [comment]
Maybe some retained setting on the MQTT server? Tricky one.
